go/service/chat_boxer.go

```diff
@@ -142,9 +142,60 @@ func (b *chatBoxer) boxMessageWithKey(msg keybase1.MessagePlaintext, key *keybas
 		KeyGeneration: key.KeyGeneration,
 	}
 
+	// sign the header, encrypted body
+	if err := b.signMessageBoxed(&boxed, signingKeyPair); err != nil {
```

Comment:
Well not absolutely necessary. But the NewComversationRemote2 now accepts a first message, which needs to be encrypted and signed, meaning PostLocal is not the only place that needs to sign anymore. I thought we could move them to boxer as well, since the signature verification is there too (IIRC; sorry on mobile now). It's better to be consistent: boxing is to sign and encrypt, and unboxing is to decrypt and verify signature